packages/atomic/src/components/atomic-result/readme.md

```diff
@@ -7,11 +7,11 @@
 
 ## Properties
 
-| Property               | Attribute | Description                    | Type                     | Default     |
-| ---------------------- | --------- | ------------------------------ | ------------------------ | ----------- |
-| `content` _(required)_ | `content` | The result content to display. | `string`                 | `undefined` |
-| `engine` _(required)_  | --        | The Headless Engine.           | `Engine<SearchAppState>` | `undefined` |
-| `result` _(required)_  | --        | The result item.               | `Result`                 | `undefined` |
+| Property               | Attribute | Description                    | Type                                                    | Default     |
+| ---------------------- | --------- | ------------------------------ | ------------------------------------------------------- | ----------- |
+| `content` _(required)_ | `content` | The result content to display. | `string`                                                | `undefined` |
+| `engine` _(required)_  | --        | The Headless Engine.           | `CoreEngine<SearchAppState, SearchThunkExtraArguments>` | `undefined` |
```

Comment:
🤔  Isn't that by definition a breaking change ?

If by doing changes in headless it caused the generated doc from atomic to change the types/interface, this should raise a red flag...

It means something subtle has changed from an API point of view.

Or maybe it's just a red herring !

Reply:
Yep, that's a fair point. It's not breaking per say, but I did define `Engine` as a `type` instead of an `interface`. Atomic is seeing this and resolving the type. I will switch to an interface and regenerate the docs.